custom/inddex/fixtures.py

```diff
@@ -34,4 +33,6 @@ def get_recipes(self):
         for item in iter_fixture_items_for_data_type(self.domain, recipes_data_type.get_id):
             item_dict = {field_name: field_list.field_list[0].field_value
                          for field_name, field_list in item.fields.items()}
-            yield RecipeIngredient(**item_dict)
+            # The fixture contains duplicate entries for each language.
+            if item_dict.pop('iso_code') == 'en':
```

Comment:
Is it safe to assume that it is always English that we want?

Reply:
So far, I think so.